src/Files.App/Views/Properties/GeneralPage.xaml.cs

```diff
@@ -158,6 +165,7 @@ async Task<bool> SaveBaseAsync(ListedItem item)
 
 		public override void Dispose()
 		{
+			_updateDateDisplayTimer.Stop();
 		}
```

Comment:
I think we should also unsubscribe the `Tick` event here

Reply:
I believe unsubscribing is not needed because both the timer and the event are private members of the page and they have the same lifecycle.